src/client/game.cpp

```diff
@@ -493,7 +497,7 @@ class GameGlobalShaderConstantSetter : public IShaderConstantSetter
 			sunlight.b };
 		m_day_light.set(dnc, services);
 
-		u32 animation_timer = porting::getTimeMs() % 100000;
+		u32 animation_timer = porting::getTimeMs() % 1000000;
 		float animation_timer_f = (float)animation_timer / 100000.f;
```

Comment:
shouldn't this also be adjusted so animationTimer stays within `[0.0, 1.0[`?

Reply:
That would slow down any current animation (and hence would require a shader fix). And then when you speed up the timer there by 10x, you'd get the wrap exactly after 100s again.

The only way is to separate the wrap from the speed. The speed stays the same, but it wrap 10x slower.

Shader do not (and should not) rely on specific values.

Reply:
Makes sense.